docs/markdown/Cython.md

```diff
@@ -60,3 +60,24 @@ py.extension_module(
     dependencies : dep_py,
 )
 ```
+
+## Limited API support
+
+*(New in 1.4)*
+
+Meson version 1.4 extends the support of the `limited_api` keyword to Cython.
+
+```meson
+project('my project', 'cython')
+
+py.extension_module(
+    'foo',
+    'foo.pyx',
+    dependencies : dep_py,
+    limited_api : '3.8'
+)
+```
+
+Cython's support for Python's Limited API was introduced in Cython version 3.0.
+If Meson detects that the user's Cython compiler is not greater than or equal to
+version 3.0, a warning is issued.
```

Comment:
We *should* note that here, though, yes.


"If the cython version is too old, meson issues a warning but proceeds" may be a better phrasing. The key information here that users care about is that it is safe/allowable to request the limited API without checking the cython version.

At least from a documentation standpoint.

Reply:
Thanks, I wasn't sure about the wording here at all. I'll update to your suggestion.